extension/src/popup/components/accountHistory/AssetNetworkInfo/index.tsx

```diff
@@ -60,15 +61,20 @@ export const AssetNetworkInfo = ({
     if (networkIconUrl || assetType === "native") {
       return <img src={networkIconUrl || StellarLogo} alt="Network icon" />;
     }
-    return <div className="AssetNetworkInfo__network__icon" />;
+    return null;
```

Comment:
I can't remember the intention behind this but it seemed weird, especially in the case where you have no home domain it was more or less a floating circle with no context.

Reply:
So this is used when we can't find an icon for an asset. Maybe this should only return `null` if there is no home domain? 
<img width="358" alt="Screenshot 2024-03-22 at 5 37 32 PM" src="https://github.com/stellar/freighter/assets/6789586/ab36c0e1-8446-4846-85e5-2a8b8e9e7b80">

Reply:
Makes sense, added that check in 4b135e5b899b32ec2040d54ce0438b559fe3b135